src/Aspire.Hosting.AWS.DynamoDBLocal/Aspire.Hosting.AWS.DynamoDBLocal.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <Description>Azure resource types for .NET Aspire.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Aspire.Hosting\Aspire.Hosting.csproj" />
+    <PackageReference Include="AWSSDK.DynamoDBv2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="C:\gitrepos\aspire-workspace\aspire\THIRD-PARTY-NOTICES.TXT" />
```

Comment:
Why?

Reply:
Removed. Not sure how that got in there.